src/main/java/kr/codesqaud/cafe/controller/UserController.java

```diff
@@ -0,0 +1,57 @@
+package kr.codesqaud.cafe.controller;
+
+import kr.codesqaud.cafe.user.MemoryUserRepository;
+import kr.codesqaud.cafe.user.User;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+@Controller
+public class UserController {
+
+    private final MemoryUserRepository memoryUserRepository;
+    private final Logger logger = LoggerFactory.getLogger(UserController.class);
+
+    public UserController(MemoryUserRepository memoryUserRepository) {
+        this.memoryUserRepository = memoryUserRepository;
+    }
+
+    @GetMapping("/user/form")
+    public String join() {
+        return "user/form";
+    }
+
+    @PostMapping("/user/form")
+    public String post(@RequestParam("userId") String userId,
+                       @RequestParam String password,
+                       @RequestParam String name,
+                       @RequestParam String email) {
+
+        User user = new User(userId, password, name, email);
+        memoryUserRepository.save(user);
+        logger.info(user.toString());
```

Comment:
```suggestion
        logger.info(user);
```

이렇게 해도 의도한대로 동작하긴 합니다만, 좀 더 의미있는 로그를 남겨주세요.
왜 동작하는지도 살펴보세요.